tests/roles/dataplane_adoption/defaults/main.yaml

```diff
@@ -84,6 +84,25 @@ edpm_computes: |-
 # The variables inside edpm_network_config_template are evaluated based
 # on the OpenstackDataplaneNodeSet inventory. For adding a variable to the invetory,
 # please add it to OpenstackDataplaneNodeSet.nodeTemplate.ansibleVars
+
+edpm_bootstrap_command: |
+              # This is a hack to deploy RDO Delorean repos to RHEL as if it were Centos 9 Stream
```

Comment:
Let's use the same solution for indentation as edpm_network_config_template uses? So that we don't have to put this seemingly random number of spaces at each line?